### DISCUSSION TITLE
How to run Automate scripts from termux?

### DISCUSSION THREAD

**Discussion by ZaxonXP** [termux/termux-app#3252] [Q&A]
I have some scripts made in Automate application. I wonder how can I run them from Termux?
Scripts has the shortcut icons on the home screen? Also, if possible, I would like to know how to run them without creating desktop shortcut

Kind regards,
Piotr

**Comment by jothi-prasath** [comment]
you can use cron job  https://www.reddit.com/r/termux/comments/i27szk/comment/g02pghj/?utm_source=share&utm_medium=web2x&context=3

**Reply by ZaxonXP** [reply]
I meant [Automate Android APK](https://play.google.com/store/apps/details?id=com.llamalab.automate&pli=1) and the scripts made in it.

**Comment by KebabLord** [comment]
i guess you can use tasker+termux-tasker as a bridge: https://f-droid.org/tr/packages/com.termux.tasker/

This can run stuff in Termux and obtain it's output for use in other apps. Other apps can also trigger Tasker to run these commands and provide output to somewhere your other apps can use. So it would make a great bridge in your use-case i guess.
